modules/ovn-kubernetes-architecture-con.adoc

```diff
@@ -12,16 +12,16 @@ The key components are:
 * **OVN Southbound database (`sbdb`)** - Stores the physical and logical network configuration state for OpenVswitch (OVS) system on each node, including tables that bind them.
-* **ovn-northd** - This is the intermediary client between `nbdb` and `sbdb`. It translates  the logical network configuration in terms of conventional network concepts, taken from the `nbdb`, into  logical data path flows in the `sbdb` below it. The container name is `northd` and it runs in the `ovnkube-master` pods.
+* **ovn-northd** - This is the intermediary client between `nbdb` and `sbdb`. It translates  the logical network configuration in terms of conventional network concepts, taken from the `nbdb`, into  logical data path flows in the `sbdb` below it. The container name is `northd` and it runs in the `ovnkube-node` pods.
 * **ovn-controller** - This is the OVN agent that interacts with OVS and hypervisors, for any information or update that is needed for `sbdb`. The `ovn-controller` reads logical flows from the `sbdb`, translates them into `OpenFlow` flows and sends them to the node’s OVS daemon. The container name is `ovn-controller` and it runs in the `ovnkube-node` pods.
 
+The OVN northd, northbound database and southbound database run on each node in the cluster and mostly contain and process information that is local to that node.
+
 The OVN northbound database has the logical network configuration passed down to it by the cloud management system (CMS).
 The OVN northbound Database contains the current desired state of the network, presented as a collection of logical ports, logical switches, logical routers, and more.
 The `ovn-northd` (`northd` container) connects to the OVN northbound database and the OVN southbound database.
 It translates the logical network configuration in terms of conventional network concepts, taken from the OVN northbound Database, into logical data path flows in the OVN southbound database.
 
-The OVN southbound database has physical and logical representations of the network and binding tables that link them together. Every node in the cluster is represented in the southbound database, and you can see the ports that are connected to it.
-It also contains all the logic flows, the logic flows are shared with the `ovn-controller` process that runs on each node and the `ovn-controller` turns those into `OpenFlow` rules to program `Open vSwitch`.
+The OVN southbound database has physical and logical representations of the network and binding tables that link them together. It contains the chassis information of the node and other constructs like remote transit switch ports that are required to to connect to the other nodes in the cluster. The OVN southbound database also contains all the logic flows, the logic flows are shared with the `ovn-controller` process that runs on each node and the `ovn-controller` turns those into `OpenFlow` rules to program `Open vSwitch`.
```

Comment:
```suggestion
The OVN southbound database has physical and logical representations of the network and binding tables that link them together. It contains the chassis information of the node and other constructs like remote transit switch ports that are required to to connect to the other nodes in the cluster. The OVN southbound database also contains all the logic flows. The logic flows are shared with the `ovn-controller` process that runs on each node and the `ovn-controller` process turns those into `OpenFlow` rules to program `Open vSwitch`.
```
Is there a noun that can describe what `Open vSwitch` is?

Reply:
It is a network switch as far as I understand. I _think_ adding that after `OvS` sounds awkward though. Let me check with SME and see if we can come up with something better.